source/templates/handlebars-basics.md

```diff
@@ -1,64 +1,72 @@
-automatically for you and is displayed when your app loads.
-
-Next, you can define templates in the `app/templates` folder. By default,
-a route will render a template with the same name as the route.
+Templates in an Ember CLI app are usually stored in the `app/templates` folder.
+By default, a route will render a template with the same name as the route. For
+example this template would render for the `kittens` route:
 
 ```app/templates/kittens.hbs
 <h1>Kittens</h1>
 <p>Kittens are the cutest!</p>
 ```
 
-If you would like to create a template that is shared across many areas of your site, you should investigate [components](../../components/defining-a-component/).
+The `app/templates/application.hbs` file is the main template for your
+application. If you have a new Ember app, you can change the contents of that
+file and that content should appear in the browser. As a new user, you
+may want to experiment with content in the application template before diving
+in to routing.
+
+If you want to learn more about how routes and templates work together in
+Ember, jump to
+[The Application Route](../../routing/defining-your-routes/#toc_the-application-route)
+section of the guides.
 
-### Handlebars Expressions
+### Displaying Properties
 
-Each template has an associated _controller_: this is where the template
-finds the properties that it displays.
+Templates are backed with a context. A context is an object from which
+mustaches read their properties. In Ember this is often a component. For
+templates rendered by a route (like `application.hbs`), the context is a
+controller.
 
-You can display a property from your controller by wrapping the property
-name in curly braces, like this:
+For example, this `application.hbs` template will render a first and last name:
 
-```handlebars
+```app/templates/application.hbs
 Hello, <strong>{{firstName}} {{lastName}}</strong>!
 ```
 
-This would look up the `firstName` and `lastName` properties from the
-controller, insert them into the HTML described in the template, then
-put them into the DOM.
+The `firstName` and `lastName` properties are read from the
+context (the application controller in this case), and rendered inside the
+`<strong>` HTML tag.
 
-By default, your top-most application template is bound to your application controller. Note that this file is not shown by default because it is created behind the scenes by Ember CLI. To customize the controller, create the following file:
+To provide a `firstName` and `lastName` to the above template, properties
+must be added to the application controller. If you are following along with
+and Ember CLI application, you may need to create this file:
 
 ```app/controllers/application.js
+import Ember from 'ember';
+
 export default Ember.Controller.extend({
-  firstName: "Trek",
-  lastName: "Glowacki"
+  firstName: 'Trek',
+  lastName: 'Glowack'
```

Comment:
I don't think we have agreement about single vs double quotes. I'd love for that to get added to https://github.com/emberjs/ember.js/blob/master/STYLEGUIDE.md and then we change it everywhere in the Guides with a single PR.